forecast_figure_scratch.R

```diff
@@ -0,0 +1,157 @@
+library(tidyverse)
+library(fs)
+library(arrow)
+library(hewr)
+library(forecasttools)
+pyrenew_hew_config_dir <- path("private_data/pyrenew-hew-config")
+prism_thresholds <- read_parquet(path(pyrenew_hew_config_dir, "prism_thresholds", ext = "parquet")) # nolint
+model_run_dir <- path("~/pyrenew-hew/private_data/pyrenew-test-output/influenza_r_2024-11-27_f_2024-08-24_t_2024-11-21/model_runs/CA") # nolint
+
+parsed_model_run_dir <- parse_model_run_dir_path(model_run_dir)
+
+
+disease_name <- parsed_model_run_dir$disease
+data_vintage_date <- parsed_model_run_dir$report_date
+
+target_disease <- "prop_disease_ed_visits"
+forecast_ci <- read_parquet(path(model_run_dir, "forecast_ci", ext = "parquet"))
+combined_dat <- read_parquet(path(model_run_dir, "combined_training_eval_data",
+  ext = "parquet"
+))
+
+y_transform <- "identity"
+
+
+horizon_weeks <- 1
+highlight_forecast_date <- epiweek_to_date(
+  epiweek = epiweek(data_vintage_date),
+  epiyear = epiyear(data_vintage_date),
+  day_of_week = 7
+) + horizon_weeks * 7
+
+prism_color_scale <- c(
+  Minimal = "#D7F2ED", Low = "#B8E5AC", Moderate = "#FEA82F",
+  High = "#F05C54", `Very High` = "#A03169", `Data Unavailable` = "#EBEBEB"
+)
+
+
+get_prism_thresholds <- function(disease, state_abb) {
```

Comment:
waiting on https://github.com/CDCgov/forecasttools/pull/31

Reply:
this should now be unblocked @damonbayer. Thanks for your patience!